### PR TITLE
Start of IRS Report

### PR DESCRIPTION
**Pull request by awolfe76**
This is the start of the IRS Report page (see #46). A later PR will finish it, with tests. Wanted to send this PR first though because of the changes to the states.
- adds new states for IRS creation and verification
  - creation state needed because the report will be creating on-the-fly after all other verifications are complete
- creates generic `IRSReport.jsx` component
- changes to `.json` and `.jsx` files to demonstrate
- route updates and additions

### CODE REVIEW

File: src/js/SubmissionContainer.jsx [hunk 1]
```diff
@@ -57,11 +60,18 @@ var SubmissionContainer = React.createClass({
 
     if(code > 5) editsContainer = <EditsContainer/>
 
-    if(code > 8){
+    if(code > 9){
+      irs = <IRS/>;
+      uploadForm = null;
```
Comment (wpears) [inline src/js/SubmissionContainer.jsx]: I don't think we want `null` here (or below if `code > 10`). At least how I've been thinking about it, we'll just keep adding new components onto the page as the user move through the process. This way they'll know precisely where they are in the process, etc.

**Comment by wpears** [comment]
@awolfe76 Just the one comment above, otherwise seems good for the next PR

File: src/js/SubmissionContainer.jsx (hunk 1 above)
Comment (awolfe76) [inline src/js/SubmissionContainer.jsx]: Hmm, ok. I think I see what you mean. Basically just leave all the components there once they are added? And let them render however they'll render ... could be empty even but that's ok.

**Comment by awolfe76** [comment]
Updates added. Mind giving it another quick look, at least at [`SubmissionContainer`](https://github.com/cfpb/hmda-platform-ui/pull/112/files#diff-9ceb5c7e53f49f44cd899504ec565cf1).

**Comment by wpears** [comment]
🚀

**Pull request merged**